manifests/master/puppetdb_conf.pp

```diff
@@ -30,9 +31,16 @@
       value   => $port,
     }
   } else {
-    ini_setting { 'puppetdbserver_urls':
-      setting => 'server_urls',
-      value   => "https://${server}:${port}/",
+    unless $servers {
+      ini_setting { 'puppetdbserver_urls':
+        setting => 'server_urls',
+        value   => "https://${server}:${port}/",
+      }
+    } else {
+      ini_setting {'puppetdbserver_urls':
+        setting => 'server_urls',
+        value   => join($servers, ',')
+      }
```

Comment:
Yeah disregard this comment, order does matter here as the first PuppetDB in the list is the first to get queried :)